.gitignore

```diff
@@ -3,5 +3,7 @@
 Builds/LinuxMakefile/*
 Builds/TestFiles/*
 Builds/VisualStudio2022/*
+Builds/MacOSX/*
 JuceLibraryCode/*
 Reaper/*
+.DS_Store
```

Comment:
Missing newline at end of file

Reply:
resolved

Reply:
Yeah, it's one of my pet peeves with git. Adding it to an ignore should mark it for removal in the same commit imo. Unfortunately I've always worked around it by manually marking for removal and then renaming the files on my machine before submit (and putting them in a separate directory that git can't see). It's gross.

Reply:
Okay this seems to require even more time than expected. Will cleanup later.